### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by mentalitybl**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by just-ero** [comment]
@FromDarkHell Do you approve of these changes?

**Comment by just-ero** [comment]
I for one don't. If this script receives an update, this link needs to be updated once again as well.

@mentalitybl, please change this link to something permanent, such as https://gist.github.com/mentalitybl/9ede8a851bb7683089b43a3d723594ef/raw/bl1e.asl.

**Comment by FromDarkHell** [comment]
I'm fine with the changes, should probably be a more permanent link though 👍

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -3219,10 +3219,10 @@
             <Game>BL1GOTY</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/FromDarkHell/Livesplit.LoadRemovers/master/Borderlands%201%20Enhanced%20Edition/BLGOTY.asl</URL>
+            <URL>https://gist.githubusercontent.com/mentalitybl/9ede8a851bb7683089b43a3d723594ef/raw/8d514b62a6ddb172bfda9db0efd28db4d16ceba1/bl1e.asl</URL>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:3222]:
```suggestion
            <URL>https://gist.github.com/mentalitybl/9ede8a851bb7683089b43a3d723594ef/raw/bl1e.asl</URL>
```

**Pull request merged**